extensions/binding/org.eclipse.smarthome.binding.weatherunderground/ESH-INF/thing/thing-types.xml

```diff
@@ -58,7 +58,7 @@
 				<label>API Key</label>
 				<description>API key to access the Weather Underground service</description>
 			</parameter>
-			<parameter name="location" type="text" required="true">
+			<parameter name="location" type="text" required="true" pattern="(^-?[0-9]{1,2}\.[0-9]+,-?[0-9]{1,2}\.[0-9]+$)|(^([A-Z]|[a-z])+\/([A-Z]|[a-z])+$)|(^([A-Z]|[a-z])+$)|(^pws:([A-Z]|[a-z]|[0-9])+$)">
```

Comment:
The regex doesn't even match all of the examples from the readme, so maybe it's better to drop it again?